addons/mail/static/src/core/common/store_service.js

```diff
@@ -103,6 +103,7 @@ export class Store extends BaseStore {
     inPublicPage = false;
     odoobot = Record.one("Persona");
     users = {};
+    groupIdToPersonas = new Map();
```

Comment:
To remove (it's replaced by `ResGroups/personas`:
```js
// in new file res_groups_model.js, model "ResGroups":
personas = Record.many("Persona");
```
_Note that you don't need to define inverse: it's implied by defining the inverse on the other field._